fix(plugin-express): fix double span end #908 (#910)
open-telemetry/opentelemetry-js: packages/opentelemetry-plugin-express/README.md

See [examples/express](https://github.com/open-telemetry/opentelemetry-js/tree/master/examples/express) for a short example.

### Caveats

> Because of the way express works, it's hard to correctly compute the time took that each span took to complete. For this reason the time you'll see reported for each span will **not** represent the time it really took.

Reviewer: Only for asynchronous spans

Author: I don't why but at some point i though that it wasn't possible for sync callback too so i'd removed the acurate reporting for all spans, but you are right. I've repushed the previous implementation that report accurate duration for sync middlewares

> Because of the way express works, it's hard to correctly compute the time took by asynchronous code to complete. For this reason the time you'll see reported for asynchronous middlewares and request handlers will **not** represent the time it really took.

Reviewer: ```suggestion
Because of the way express works, it's hard to correctly compute the time taken by asynchronous middlewares and request handlers. For this reason, the time you'll see reported for asynchronous middlewares and request handlers will only represent the synchronous execution time, and **not** any asynchronous work.
```
